src/main/java/taskbot/command/Command.java

```diff
@@ -0,0 +1,38 @@
+package taskbot.command;
+
+import taskbot.exceptions.TaskbotException;
+import taskbot.task.TaskList;
+
+/**
+ * Encapsulates the commands the user
+ * can give TaskBot.
+ */
+public abstract class Command {
+    // Flag for the ExitCommand
+    private final boolean isExit;
+
+    /**
+     * Creates a Command.
+     *
+     * @param isExit Signifies whether it is an ExitCommand.
+     */
+    public Command(boolean isExit) {
+        this.isExit = isExit;
+    }
+
+    /**
+     * Performs the relevant operation tied to the Command type on the given task list.
```

Comment:
I think this part explained the purpose of the code really well

Reply:
Thanks for your time and feedback! :)